site/en/docs/devtools/console/utilities/index.md

```diff
@@ -86,7 +86,7 @@ overwritten, and `$` will correspond to that library's implementation.
 
 {% endAside %}
 
-## \$\$(selector, \[startNode\]) {: #queryselectorall }
+## \$\$(selector, \[startNode\]) {: #querySelectorAll-function }
```

Comment:
Don't our linter rules warn/error on camelCase IDs?

Reply:
No, and I'm not fussed about this right now. I think using their full cased name is fine—we have cased pages.